<ui version="4.0" >
 <class>TrainingSettings</class>
 <widget class="QWidget" name="TrainingSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" colspan="3" >
    <widget class="QComboBox" name="stopFunctionCombo" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_44" >
     <property name="text" >
      <string>Desired error</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QComboBox" name="errorFunctionCombo" />
   </item>
   <item row="9" column="2" colspan="2" >
    <widget class="QLineEdit" name="minWeightEdit" >
     <property name="text" >
      <string>-0,1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Error function</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Stop function</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3" >
    <widget class="QDoubleSpinBox" name="learningMomentumSpin" >
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QLabel" name="label_41" >
     <property name="text" >
      <string>Minimum random weight</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3" >
    <widget class="QDoubleSpinBox" name="learningRateSpin" >
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>99.900000000000006</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>0.700000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QComboBox" name="trainingAlgorithmCombo" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_43" >
     <property name="text" >
      <string>Traing epochs</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_25" >
     <property name="text" >
      <string>Training algorithm</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_26" >
     <property name="text" >
      <string>Learning rate</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3" >
    <widget class="QPushButton" name="randomizeButton" >
     <property name="text" >
      <string>Randomize</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2" >
    <widget class="QLabel" name="label_42" >
     <property name="text" >
      <string>Maximum random weight</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QLineEdit" name="trainingEpochsEdit" >
     <property name="inputMask" >
      <string/>
     </property>
     <property name="text" >
      <string>1000</string>
     </property>
     <property name="maxLength" >
      <number>25</number>
     </property>
    </widget>
   </item>
   <item row="11" column="2" >
    <widget class="QPushButton" name="initializeWeightsButton" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>Initialize</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2" >
    <widget class="QLineEdit" name="maxWeightEdit" >
     <property name="text" >
      <string>0,1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="QLineEdit" name="errorLimitEdit" >
     <property name="text" >
      <string>0.0035</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_27" >
     <property name="text" >
      <string>Learning momentum</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QCheckBox" name="cascadeTrainingCheck" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>CascadeTraining</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
